news/_posts/2020-02-04-the-new-prestashop-test-framework.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "The (new) PrestaShop Test Framework"
+subtitle: "And how we’re continually improving the quality of the PrestaShop project"
+date:  2020-02-05 12:00:00
+authors: [SimonGarny]
+icon: icon-checked
+image: /assets/images/theme/meta-logo-build.png
+tags: [QA, tests, community]
+---
+
+For a few months, quality has been a priority for PrestaShop: the QA team and I are proud to announce that we’re making great steps in improving the overall quality of the core project. Automatizing tests is a big part of this process: let’s see what we’re doing, why, and how !
```

Comment:
```suggestion
For a few months, quality has been a priority for PrestaShop: the QA team and I are proud to announce that we’re making great steps in improving the overall quality of the Core project. Automatizing our tests is a big part of this process: let’s see what we’re doing, why, and how!
```

Reply:
if we can avoid "our" and "we" here this is better. what about "Automatizing the tests"